lib/ihub/appointments/response.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'common/client/concerns/service_status'
+require 'common/models/base'
+
+module IHub
+  module Appointments
+    class Response < Common::Base
+      include Common::Client::ServiceStatus
+
+      attribute :status, Integer
+      attribute :response_data, Hash
+
+      def initialize(attributes = nil)
+        super(attributes) if attributes
+        self.status = attributes[:status]
+      end
+
+      def self.from(response)
+        new(status: response.status, response_data: response.body)
+      end
+
+      def ok?
```

Comment:
We re-use a lot of these base methods in our services (ok?, cache?, response_status, etc.). At some point we should consolidate into a common place.  Perhaps iHub we should do that during the iHub integration?